### ISSUE TITLE
Close project window via main menu (Improve UX for tiling window managers (i3/sway/etc))

### ISSUE THREAD

**Issue by lebe-dev** [zed-industries/zed#19501]
### Check for existing issues

- [X] Completed

### Describe the feature

Tiling window managers typically don't have dedicated window controls (at least not all of them) to close individual windows.

![image](https://github.com/user-attachments/assets/dbe1ebb9-eddc-4579-978b-fa7e68e05765)

When I use 'Close Workspace', 'Quit', or the Alt-F4 shortcut, it closes Zed along with all open windows.

### If applicable, add mockups / screenshots to help present your vision of the feature

Just add "Close project" or "Close window" into Main menu.

**Comment by notpeter** [comment]
Yes. We should probably add `workspace::CloseWindow"` to the menu.

In the meantime as a workaround this is bound to `ctrl-shift-w` in the default keymap
https://github.com/zed-industries/zed/blob/92c29be74cc2ac09dfe0d71d5a1048121b6ab4c6/assets/keymaps/default-linux.json#L22

**Comment by lebe-dev** [comment]
> Yes. We should probably add `workspace::CloseWindow` to the menu.
> 
> In the meantime as a workaround this is bound to `ctrl-shift-w` in the default keymap
> 
> https://github.com/zed-industries/zed/blob/92c29be74cc2ac09dfe0d71d5a1048121b6ab4c6/assets/keymaps/default-linux.json#L22

`ctrl-shift-w` closes all windows of zed not just current.

**Comment by ferrao** [comment]
Not sure if it helps you @lebe-dev , but I am using the following keymap to close specific editor windows:

```{
    "context": "VimControl && !menu",
    "bindings": {
      "ctrl-w": ["pane::CloseActiveItem", { "saveIntent": "saveAll" }]
    }
  }
```

**Comment by lebe-dev** [comment]
> Not sure if it helps you @lebe-dev , but I am using the following keymap to close specific editor windows:
> 
> ```
>     "context": "VimControl && !menu",
>     "bindings": {
>       "ctrl-w": ["pane::CloseActiveItem", { "saveIntent": "saveAll" }]
>     }
>   }
> ```

Thanks, but it doesn't work in my case.

Open two editor windows with different projects. Action `pane::CloseActiveItem` just closes editor pane in the current window.

What behavior i'm expecting: close the current window not all windows.

**Comment by 0xtimsb** [comment]
This issue can be closed now that we have proper menus for Linux: https://github.com/zed-industries/zed/pull/21873
